example/todomvc-flow/src/infra/TodoListRepository.js

```diff
@@ -32,10 +32,12 @@ export class TodoListRepository extends EventEmitter {
     /**
      * @returns {TodoList|undefined}
      */
-    lastUsed(): ?TodoList {
+    lastUsed(): TodoList {
```

Comment:
Does flow has Non-null assertion operator like `!`?

- https://github.com/niieani/typescript-vs-flowtype#non-null-assertion-operator

I do this change for avoiding following error

![error](https://monosnap.com/file/vUJ4rIaPbgrvRKcbqupDfpI1V1ql7O.png)